src/components/PackageNav/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { Box, Flex, Text } from "@chakra-ui/react";
+import { Fragment } from "react";
+
+function PackageNavItem({ title, path }: PackageNavItem) {
+  return (
+    <Box>
+      <a href={path}>
+        <Text>{title}</Text>
+      </a>
+    </Box>
+  );
+}
+
+function PackageNavItems({ items }: PackageNavProps) {
+  return (
+    <Flex direction="column" paddingLeft={4}>
+      {items.map((item) => {
+        return (
+          <Fragment key={item.path}>
+            <PackageNavItem {...item} />
+            <PackageNavItems items={item.children} />
+          </Fragment>
+        );
+      })}
+    </Flex>
+  );
+}
+
+export interface PackageNavItem {
+  title: string;
+  path: string;
+  level: number;
```

Comment:
Is this actually used somewhere in this component?

Reply:
The `level` property? No, it just greatly simplifies the recursive building of the nav items. Additionally I do believe it's likely to be used in the future as the level will dictate the styling of the elements.